packages/eslint-plugin/docs/rules/prefer-readonly-parameter-types.md

```diff
@@ -129,6 +129,101 @@ interface Foo {
 
 ## Options
 
+### `allow`
+
+Some complex types cannot easily be made readonly, for example the `HTMLElement` type or the `JQueryStatic` type from `@types/jquery`. This option allows you to globally disable reporting of such types.
+
+Each item must be one of:
+
+- A type defined in a file (`{from: "file", name: "Foo", path: "src/foo-file.ts"}` with `path` being an optional path relative to the project root directory)
+- A type from the default library (`{from: "lib", name: "Foo"}`)
```

Comment:
`defaultLib` would be a better `from` value as it would make it more explicit that it refers to the TypeScript default libs rather than just a random lib.

Reply:
See https://github.com/typescript-eslint/typescript-eslint/issues/5271#issuecomment-1293768915

Reply:
Ahh, that's were all the conversation about the details is. 🙂